src/popup/router/components/Button.vue

```diff
@@ -230,7 +262,7 @@ export default {
     border-radius: variables.$border-radius-interactive;
     width: auto;
     padding: 5px 20px;
-    line-height: 20px;
+    line-height: 200%;
```

Comment:
this set always text on center

Reply:
But it brakes the button. To center something in the element please use flex and align-items.
Line height is not a good solution for aligning text because sometimes the text on the buttons can break into more than one line. In this situation the button will look bad.